front/lib/amplitude/back/generated/index.ts

```diff
@@ -75,6 +75,21 @@ export interface IdentifyProperties {
   SignupDate?: string;
 }
 
+export interface AssistantCreatedProperties {
+  assistantActionType: string;
+  assistantId: string;
+  assistantName: string;
+  assistantScope: string;
+  /**
+   * | Rule | Value |
```

Comment:
?

Reply:
When setting a property as `number` on Amplitude, it generates that.